api/src/objects/collection.rs

```diff
@@ -120,13 +120,48 @@ impl Collection {
     }
 
     /// The list of minted NFTs from the collection including the NFTs address and current owner's wallet address.
-    async fn mints(&self, ctx: &Context<'_>) -> Result<Option<Vec<CollectionMint>>> {
-        let AppContext {
-            collection_mints_loader,
-            ..
-        } = ctx.data::<AppContext>()?;
-
-        collection_mints_loader.load_one(self.id).await
+    async fn mints(
+        &self,
+        ctx: &Context<'_>,
+        after: Option<String>,
+        first: Option<i32>,
+    ) -> Result<connection::Connection<u64, u64, connection::EmptyFields, CollectionMint>> {
+        connection::query(
+            after,
+            None,
+            first,
+            None,
+            |after, before, first, last| async move {
+                const MAX_LIMIT: u64 = 32;
+
+                assert!(before.is_none());
+                assert!(last.is_none());
+                let offset = after.map_or(0, |a| a + 1);
+                let limit = first
+                    .and_then(|f| u64::try_from(f).ok())
+                    .map_or(MAX_LIMIT, |f| f.min(MAX_LIMIT));
+
+                let mut conn = connection::Connection::new(offset > 0, true);
+
+                let AppContext { db, .. } = ctx.data::<AppContext>()?;
+
+                let mints = collection_mints::Entity::find()
+                    .filter(collection_mints::Column::Id.eq(self.id))
+                    .offset(offset)
+                    .limit(limit)
+                    .all(db.get())
```

Comment:
https://www.sea-ql.org/SeaORM/docs/basic-crud/select/#cursor-pagination
sea orm has cursor based pagination helpers was there an issue in using them?

Reply:
SeaORM's pagination seems to encode the cursor as a column but it seemed like async-graphql's connection types want an index-based cursor.  I can try using it but it'll still require `.offset()` and I'm not sure how that will play with the `Cursor` struct.